do-not-commit/marshal.py

```diff
@@ -0,0 +1,10 @@
+import marshal
+
+fin = open("index.mar")
+for line in fin:
+    # ruleid: marshal-usage
+    marshal.dumps(line)
```

Comment:
The marshal module is not intended to be secure against erroneous or maliciously constructed data.
Never unmarshal data received from an untrusted or unauthenticated source.
See more details: https://docs.python.org/3/library/marshal.html?highlight=security ([dit.marshal.marshal-usage](https://semgrep.dev/r/dit.marshal.marshal-usage))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=dit.marshal.marshal-usage) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>